internal/learn/about-iota/data-transfer.md

```diff
@@ -9,96 +9,104 @@ IOTA offers the ability to transfer Data for free. The Data transfer is fast, im
 
 Clients, which can be wallets or applications, send and receive messages (data objects) through nodes in IOTA. Nodes are the entry and exit points for these messages, and those nodes communicate with each other and with the connected clients.
 
-There are several types of messages implemented in IOTA. Some messages transfer value (the IOTA token or digital assets), while others transfer only pure data, and some message types can even contain both value and data. This flexible message structure enables the decentralized transport of data and value in a single message with the highest security grade and the total absence of fees. The network nodes take care of the secure distribution of all those messages in the Tangle.
+There are several types of messages implemented in IOTA. Some messages transfer value (the IOTA token or digital assets), while others transfer only pure data, and some message types can even contain both values and data. This flexible message structure enables the decentralized transport of data and value in a single message with the highest security grade and the total absence of fees. The network nodes take care of the secure distribution of all those messages in the Tangle.
```

Comment:
As a non-English reader, this now reads as if values is meant as data values and not value as economic value.

Reply:
Ah, this was only to make it more "plural" friendly. I can rework it.